test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/EnumTypeModelBinderTest.cs

```diff
@@ -133,15 +132,15 @@ public class EnumTypeModelBinderTest
         [InlineData("8, 1", false)]
         [InlineData("Value2, Value8", false)]
         [InlineData("value8,value4,value2,value1", false)]
-        public async Task BindModel_BindsTo_NullableFlagsEnumType(string flagsEnumValue, bool allowBindingUndefinedValueToEnumType)
+        public async Task BindModel_BindsTo_NullableFlagsEnumType(string flagsEnumValue, bool suppressBindingUndefinedValueToEnumType)
         {
             // Arrange
             var modelType = typeof(FlagsEnum?);
             var enumConverter = TypeDescriptor.GetConverter(GetUnderlyingType(modelType));
             var expected = enumConverter.ConvertFrom(flagsEnumValue).ToString();
             var binderInfo = GetBinderAndContext(
                 modelType,
-                allowBindingUndefinedValueToEnumType,
+                suppressBindingUndefinedValueToEnumType,
```

Comment:
All of the tests above this line don't actually care about `suppressBindingUndefinedValueToEnumType` and test both options.